### DISCUSSION TITLE
Adjoint methods for structural optimization

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#1932] [General]
Does SU2 support adjoint methods for structural optimization？

**Comment by pcarruscag** [comment]
It does indeed.
https://github.com/su2code/SU2/tree/master/TestCases/disc_adj_fea <- Discrete adjoint of an elasticity problem
https://github.com/su2code/SU2/tree/master/TestCases/fea_topology <- Topology optimization

**Comment by pcarruscag** [comment]
https://github.com/su2code/FADO/blob/master/examples/example3_SU2/example.py <- Shape and topology optimization with multiple load cases

**Comment by Zcaic** [comment]
Thank you very much
